test/translation/test_mutant_generator.py

```
@@ -0,0 +1,133 @@
+        NegOp(OrOp(Bool(True), Bool(True))),
+        NegOp(AndOp(Bool(False), Bool(True))),
+        NegOp(AndOp(Bool(True), Bool(False))),
+    ]
+    _assert_mutants(mutants, expected_mutants)
+
+
+def test_get_single_point_mutants_requires_clause() -> None:
+    requires_clause = RequiresClause(meta=None, expr=AndOp(Bool(True), Bool(True)))
+    mutants = mutant_generator.get_single_point_mutants(requires_clause)
+    expected_mutants: list[CBMCAst] = [
+        RequiresClause(OrOp(Bool(True), Bool(True))),
```

Ruff (0.15.4)

[error] Boolean positional value in function call

(FBT003)